website/docs/verifying-images.md

```diff
@@ -0,0 +1,260 @@
+# Verifying Release Images
+
+All official Dalec container images published to GitHub Container Registry (GHCR) are cryptographically signed and include attestations for build provenance. This enables users to verify that images are authentic and haven't been tampered with.
+
+## Why Verify Images?
+
+Image verification provides several security benefits:
+
+- **Authenticity**: Confirm the image was published by the official Dalec repository
+- **Integrity**: Detect if the image has been tampered with or modified
+- **Provenance**: Verify the image was built from the expected source code and workflow
+- **Supply Chain Security**: Comply with security best practices and policies (e.g., SLSA framework)
+
+## What Gets Signed
+
+The following container images are signed when published as releases:
+
+- **Frontend image**: `ghcr.io/project-dalec/dalec/frontend` (main BuildKit frontend)
+- **Worker images**: Various target-specific worker images (e.g., `ghcr.io/project-dalec/dalec/mariner2/container`)
+
+Images are signed using [Sigstore's cosign](https://github.com/sigstore/cosign) with keyless signing via GitHub OIDC tokens.
+
+## Prerequisites
+
+To verify images, you can use either:
+
+### Option 1: Cosign (Recommended)
+
+```bash
+# Install cosign (see https://docs.sigstore.dev/cosign/installation for more options)
+brew install cosign  # macOS
+# or
+curl -O -L "https://github.com/sigstore/cosign/releases/latest/download/cosign-linux-amd64"
+sudo mv cosign-linux-amd64 /usr/local/bin/cosign
+sudo chmod +x /usr/local/bin/cosign
+```
+
+### Option 2: GitHub CLI
+
+If you have the [GitHub CLI](https://cli.github.com/) installed (v2.49.0+), you can use the built-in attestation verification:
+
+```bash
+gh attestation verify oci://ghcr.io/project-dalec/dalec/frontend:v0.9.0 --owner project-dalec
```

Comment:
Maybe we can change all references to `v0.9.0` in this doc to `v0.19` which would be the first release that is signed.